x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -465,17 +466,17 @@ export const tasks: TelemetryTask[] = [
   {
     name: 'integrations',
     executor: async ({ transportRequest }) => {
-      const apmJobs = ['*-high_mean_response_time'];
+      const apmJobs = ['apm-*', '*-high_mean_response_time'];
 
       const response = (await transportRequest({
         method: 'get',
         path: `/_ml/anomaly_detectors/${apmJobs.join(',')}`,
```

Comment:
Using `path: `/_ml/anomaly_detectors/${ML_GROUP_NAME_APM}` will return all jobs with `group=${ML_GROUP_NAME_APM}` so should also work for old jobs.